### PR TITLE
fix(type): add new testID to typescript definition file

### PR DESCRIPTION
**Pull request by doublethefish**
Adds `testID` to the typescript type information.

Test Plan:
Testing the module in a production app compiled with tsc.

### CODE REVIEW

File: types/FBLoginButton.d.ts [hunk 1]
```diff
@@ -57,6 +57,11 @@ export class LoginButton extends React.Component<{
      * View style, if any.
      */
   style?: any;
+
+  /**
+   * testID, if any.
+   */
+  testID?: string,
```
Comment (mikehardy) [inline types/FBLoginButton.d.ts:64]: Oh of course, I should have seen that yesterday when I reviewed your implementation PR. Thanks for the follow-up

I think it needs to be like this though? It looks like we are inconsistent on this point actually! Most are semicolon-terminated (correct I think) while some are commas. I wonder if our types are being tested correctly :thinking: . Do you confirm this works for you? Can you confirm it works with the semi-colon (which I think is more correct)?

```suggestion
  testID?: string;
```

**Review by mikehardy — CHANGES_REQUESTED** [review]
tiny question on semi-colon vs comma termination on typedef

**Comment by mikehardy** [comment]
Ah, this won't be caught until #95 is done most likely. The types are not checked anywhere.

File: types/FBLoginButton.d.ts (hunk 1 above)
Comment (doublethefish) [reply types/FBLoginButton.d.ts]: Good catch, thank you. 

I had tested it locally with the correct fix but somehow submitted this... hmmm.

**Review by mikehardy — APPROVED** [review]
LGTM thanks!

**Comment by mikehardy** [comment]
Not sure what is going on with commas vs semi-colons but I discovered in my work project I was actually nulling out the types here anyway, from before there were types here. I just started using the internal types after updating to the new 4.5.0 release that has this change among others, and everything seems fine right now, so this is working for me :-)

**Pull request merged**